fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +491,35 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    c_seqnum = self.get_current_seq_number(lsa)
+                    if not c_seqnum:
+                        # If LSA wasn't in the lsdb: process it normally
+                        log.debug('LSA was not in the LSDB yet')
+
+                    else:
+                        log.debug('Seqnum for LSA in lsdb: %s)', c_seqnum)
+                        log.debug('Seqnum for newly arrived LSA: %s)', new_seqnum)
+
+                        if new_seqnum == c_seqnum:
+                            if hdr.age >= MAX_LS_AGE:
+                                log.debug("Premature aging to wrap seqnum around for LSA: %s", lsa)
+                            else:
+                                log.debug("DUPLICATE seqnum for new LSA: %s", lsa)
+
+                        elif self.is_newer_seqnum(new_seqnum, c_seqnum):
+                            log.debug("NEWER seqnum received")
+
+                        else:
+                            log.debug("OLD seqnum for LSA: %s, ignoring it...", lsa)
```

Comment:
Reduce the log volume (remove starting from: if new_seqnum == c_seqnum to here, then)

```python
if not self.is_newer_seqnum(new_seqnum, c_seqnum) and new_seqnum != c_seqnum:
   # Can get the same seqnum when flushing an LSA from the domain
    log.debug("OLD seqnum for LSA, ignoring it...")
    self.queue.task_done()
    continue
```